lib/ProxySQL_Admin.cpp

```diff
@@ -98,15 +98,16 @@ pthread_mutex_t admin_mutex = PTHREAD_MUTEX_INITIALIZER;
 
 
 static char * admin_variables_names[]= {
-  (char *)"admin_credentials",
-  (char *)"stats_credentials",
-  (char *)"mysql_ifaces",
-  (char *)"telnet_admin_ifaces",
-  (char *)"telnet_stats_ifaces",
-  (char *)"mysql_ifaces",
-  (char *)"refresh_interval",
+	(char *)"admin_credentials",
+	(char *)"stats_credentials",
+	(char *)"mysql_ifaces",
+	(char *)"telnet_admin_ifaces",
+	(char *)"telnet_stats_ifaces",
+	(char *)"mysql_ifaces",
+	(char *)"refresh_interval",
 	(char *)"read_only",
 	(char *)"version",
+	(char *)"proxysql_consul_path",
```

Comment:
Will rename to proxysql_consul_script_path and will add proper documentation in the variable doc. The entire consul integration feature will be documented in the config doc.